releasenotes/notes/quantize-entire-sql-statement-7f3685243037fbab.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Updated the obfuscator to quantize the entire SQL statement rather than just the table name.
```

We have to prefix all trace-agen changes with `APM: `:
```suggestion
    APM: Updated the obfuscator to quantize digits in IDs of SQL statement in addition to table names,
    when this option is enabled.
```